src/java.desktop/share/classes/sun/java2d/marlin/MarlinProperties.java

```diff
@@ -77,6 +77,25 @@ public static int getInitialPixelHeight() {
             64);
     }
 
+    /**
+     * Return true if the profile is 'quality' (default) over 'speed'
+     *
+     * @return true if the profile is 'quality' (default), false otherwise
+     */
+    public static boolean isProfileQuality() {
+        final String key = "sun.java2d.renderer.profile";
+        final String profile = getString(key, "quality");
```

Comment:
Not for this review, but some time we need to make sure there are no incompatibilities
associated with these properties. What I really mean is a shortcut for speed that causes
rendering that violates the SE spec needs to be disabled in JDK.
I have no reason here to suppose that but we should test it some day.

Reply:
Good point, to be discussed later